src/js/common/components/Navigation/ChallengeInviteSteps.jsx

```diff
@@ -208,10 +204,6 @@ const StepOneIconAndText = styled('div')`
   text-align: center;
   width: 169px;
 
-  img {
-    content: url(${({ isActive }) => (isActive ? StepOneActiveIcon : StepOneIcon)});
-  }
-
   a {
```

Comment:
Are these "a {" styles still needed?

Reply:
Yes, we still need <a>  because < Link> renders as an <a> tag in the browser. We use it to directly style the <a> tag nested inside the StepOneIconAndText component. 
 But also, I can try to rewrite it and put those styles directly inside the < Link>. 
Would it be better to rewrite it? Or leave it as is?